lib/generators/rspec/schlepper/templates/onetime_script_spec.rb.erb

```diff
@@ -0,0 +1,9 @@
+require_relative '<%= require_task_path %>'
+
+describe <%= task_name.classify %> do
+  let(:task) { described_class.new }
+
+  it 'returns true for success' do
+    expect(task.run).to be true
```

Comment:
Interesting... the Rails generators explicitly have something Iike `expect(false).to be` to force the user to write the tests. This will do something similar because this test will never pass until the required data is present and the task class returns true.

I don't have an opinion either way, I'm just musing.